app/api/templates/templates.js

```diff
@@ -203,7 +203,7 @@ export default {
       const entitySelects = [];
       return Promise.all(
         selects.map(select =>
-          request.get(`${dbURL}/${select.content}`).then(result => {
+          request.get(`${dbHost}/${select.content}`).then(result => {
```

Comment:
DBHOST is a process.env optional value, which is then defaulted to "localhost" on standard installations.  Where is this default happening if this variable is imported directly?

Reply:
This just fixes the fact that dbURL was undefined ;)

I added the default to database.js